packages/arb-bridge-eth/test/sequencerInbox.spec.ts

```diff
@@ -444,4 +443,40 @@ describe('SequencerInbox', async () => {
       'MAX_DELAY_BLOCKS'
     )
   })
+
+  it('cannot include before max time delay', async () => {
+    const { user, inbox, bridge, messageTester, sequencerInbox } =
+      await setupSequencerInbox(10, 100)
+    const delayedTx = await sendDelayedTx(
+      user,
+      inbox,
+      bridge,
+      messageTester,
+      1000000,
+      21000000000,
+      0,
+      await user.getAddress(),
+      BigNumber.from(10),
+      '0x1010'
+    )
+
+    const maxDelayBlocks = (await sequencerInbox.maxDelayBlocks()).toNumber()
+    // mine a lot of blocks - but use a short time per block
+    // this should mean enough blocks have passed, but not enough time
+    await mineBlocks(maxDelayBlocks + 1, 5)
+
+    await forceIncludeMessages(
```

Comment:
hm we could maybe simplify this interface by just expecting the delayedTx object (could use something like `ReturnType<sendDelayedTx>` (not sure if this is valid syntax, but general idea)

Reply:
Hmm, I think i wont - the interfaces dont match up exactly I kinda want the force include params to mirror the names in the smart contract

Reply:
yup sounds fair